memsql-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -e
+
+if [ "$1" = "memsqld" ]; then
+    # Start up the cluster
+    memsql-ops start
+    memsql-ops memsql-start --all
+    memsql-ops memsql-list
+
+    # Tail the logs to keep the container alive
+    exec tail -F /memsql/master/tracelogs/memsql.log
```

Comment:
I started with multiple files there, but it felt very noisy, so I cut back to one. Could go either way on the UX here.